.github/actions/occt-install-dep/action.yml

```diff
@@ -54,7 +54,7 @@ runs:
       if: steps.cache-occt.outputs.cache-hit != 'true'
       working-directory: ${{github.workspace}}/dependencies
       shell: bash
-      run:
+      run: |
```

Comment:
is it a fix?

Reply:
It seems to work without it but it's more consistent with it

Reply:
I think `mkdir occt_install` was probably ignored, but it's not a big deal since the directory is created automatically by CMake during the installation step.  
Ideally, `mkdir occt_build` shouldn't be necessary if we use `cmake -B x -S x` so we can remove this job step entirely.  
Not related to this PR tho, we can do it later.

Reply:
good point!